trunk/source/WitnessParser/src/de/uni_freiburg/informatik/ultimate/witnessparser/YamlWitnessParser.java

```diff
@@ -92,6 +108,19 @@ private static Stream<WitnessEntry> parseWitnessEntry(final Map<String, Object>
 		}
 	}
 
+	private static GhostVariable parseGhostVariable(final Map<String, Object> entry) {
+		return new GhostVariable((String) entry.get("name"), (String) entry.get("initial"), (String) entry.get("scope"),
+				(String) entry.get("type"));
```

Comment:
This seems similar but slightly different from the `case GhostVariable.NAME` code above. What is the reason to have both? Does it have to do with different versions of the format?

Reply:
Yes, exactly, this function is used for the format version 2.1 (therefore this function is called in the `case "ghost_instrumentation"`), while the one above is used for version 0.1. The slight difference is `variable` vs. `name` as field name for the name of the ghost variable.

Reply:
See above: It seems like a lot of effort just to keep supporting a draft format version proposed in a workshop talk. If we update our artifact repo for the workshop to point to the ULTIMATE version where this format is supported, I think we can safely abandon the old format going forward.

Reply:
Since the format was now adapted and Goblint also supports the new format, I removed ghost variables in version 0.1.